tests/locallib_exposed.php

```diff
@@ -0,0 +1,77 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Exposed class for PHPUnit test
```

Comment:
Reicht die Beschreibung
  `Exposed class for PHPUnit test`
?

Ich würde als Kommentar noch reinschreiben, wofür die Klasse ist.
Irgendwie so:
  `Class that exposes methods, i.e. allows to access protected or private mehtods, for unit testing purpose.`
Am besten mit einem Link.